docs/index.md

```diff
@@ -2,7 +2,7 @@
 
 PyDARN is an open source python library for 
 data visualization of SuperDARN data.
-support of Input/Output reading of SuperDARN DMAP format files. 
+PyDARN also includes support of Input/Output reading of SuperDARN DMAP format files via the io package [pyDARNio](https://pydarnio.readthedocs.io/en/latest/). 
```

Comment:
```suggestion
pyDARN includes support of Input/Output reading of SuperDARN DMAP and Borealis format via the io package [pyDARNio](https://pydarnio.readthedocs.io/en/latest/). 
```
py needs to be the lower case as this is how the name is spelled... keeps consistency... and something Kathryn told me to do lol